src/inspector_socket.cc

```diff
@@ -48,16 +49,17 @@ static void dump_hex(const char* buf, size_t len) {
 }
 #endif
 
+static void remove_from_beginning(std::vector<char>* buffer, size_t count) {
```

Comment:
I don’t feel too strongly about it. I’d personally have opted for a non-const reference, but if you or anyone else prefers it this way, that’s definitely okay too.